concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -232,13 +252,13 @@ def test_case_2c_wrong_timestamps(cluster_consts, cluster_url, test_id):
         force_subtask_results(
             timestamp=timestamp_to_isoformat(current_time),
             ack_report_computed_task=ack_report_computed_task(
-                timestamp=timestamp_to_isoformat(current_time - (SUBTASK_VERIFICATION_TIME * 1.4)),
+                timestamp=timestamp_to_isoformat(current_time),
                 report_computed_task=report_computed_task(
                     task_to_compute=create_signed_task_to_compute(
-                        timestamp=timestamp_to_isoformat(current_time - SUBTASK_VERIFICATION_TIME * 1.5),
+                        timestamp=calculate_timestamp(current_time, cluster_consts),
                         task_id=task_id,
                         subtask_id=subtask_id,
-                        deadline=current_time - (SUBTASK_VERIFICATION_TIME * 20),
+                        deadline=current_time - (1 + ((20 * cluster_consts.concent_messaging_time) + 183)),
```

Comment:
Could you please use a new function here? It is hard do figure out why such numbers...

Reply:
This one calculates a different value. If you look at test description and this calculation it becomes clear.

Reply:
would it be possible to have it like ~:
`deadline=calculate_deadline(current_time, cluster_consts.concent_messaging_time) + SOME_OFFSET`? Or even extract it to function: `calculate_wrong_timestamp` or sth similar?
"readibility counts" ;) (https://www.python.org/dev/peps/pep-0020/)